packages/core/src/routes/session/username-password.ts

```diff
@@ -94,8 +94,8 @@ export default function usernamePasswordRoutes<T extends AnonymousRouter>(
         passwordEncrypted,
         passwordEncryptionMethod,
         roleNames,
+        lastSignInAt: Date.now(),
       });
-      await updateLastSignInAt(id);
```

Comment:
is this function still in use?

Reply:
Yes. In some sign-in cases, we do not update other users information in DB but only the `lastSignInAt` field. Or we can simply delete this method and use `updateUserById`. I'm good with both.

Reply:
Only use `updateUserById` seems good enough to me

Reply:
use `updateUserById ` instead of all `updateLastSignInAt`.